apps/web/constants/contracts/index.ts

```diff
@@ -23,13 +23,15 @@ type RPCSubgraphAddr = {
     rpcUrl?: string;
     subgraphUrl: string;
     strategyTemplate?: Address;
+    passportScorer?: Address;
   };
 };
 
 let subgraphAddresses: RPCSubgraphAddr = {
   [localhost.id as number]: {
     rpcUrl: "http://127.0.0.1:8545",
     subgraphUrl: "http://localhost:8000/subgraphs/name/kamikazebr/gv2",
+    passportScorer: "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707",
```

Comment:
This is a just reminder gate to make it for other chains once you will deploy the contracts.

Reply:
Unresolved it because still not there for other chains. The goal would be for lucho's PR to be tested with other test chains before being merged.